deployment/helm/templates/webhook-configuration.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if .Values.webhookConfiguration.enabled }}
+apiVersion: v1
+kind: List
+metadata:
+  name: {{ .Values.webhookConfiguration.name }}
+items:
+  - apiVersion: v1
+    kind: ServiceAccount
```

Comment:
should these be surrounded by `{{- if rbac.enabled }}`?

should the rbac items be in the templates/rbac.yaml (maybe a future consideration, not for this PR)?

Reply:
I remain unclear whether this is needed with the webhook; it's unclear to me what permissions it should have

Reply:
fair ... let's leave it as is for now.  if we get some automated tests, that will make validating what is and is not needed easier :)